GliaWidgets/Public/Glia/Glia.Deprecated.swift

```diff
@@ -1,6 +1,22 @@
 import SalemoveSDK
 
 extension Glia {
+    /// Deprecated.
+    @available(*, deprecated, message: "Use configure(with:queueId:visitorContext:uiConfig:assetsBuilder:completion:) instead.")
```

Comment:
Do we have a ticket for making it unavailable in 6+ months? If not, then it has to be created to avoid piling up deprecated methods.

Reply:
Done https://glia.atlassian.net/browse/MOB-1912